Core/src/Detector/detail/CuboidalDetectorHelper.cpp

```diff
@@ -376,7 +376,7 @@ Acts::Experimental::detail::CuboidalDetectorHelper::xyzBoundaries(
   }
 
   for (auto [im, map] : enumerate(valueMaps)) {
-    for (auto [key, value] : map) {
+    for (auto [key, _] : map) {
```

Comment:
I believe the canonical way of doing is with `std::ignore`.

Reply:
I'm not aware this is canonical. The original proposal called `std::ignore` _awkward_.

Reply:
I just did a quick search:
 1x `std::ignore`
10x `[_, `
10x `, _]`

Reply:
I was actually dead wrong on this, `std::ignore` cannot be used in structured bindings at all (only in `std::tie`). Sorry for the confusion!